otter/convergence/service.py

```diff
@@ -115,6 +115,7 @@ def execute_convergence(tenant_id, group_id, log,
     desired_group_state = get_desired_group_state(
         group_id, launch_config, group_state.desired)
     steps = plan(desired_group_state, servers, lb_nodes, now)
+    yield Effect(BoundLogIntent(log, log_plan(steps)))
```

Comment:
It seems like the BoundLogIntent should be wrapped at a much higher level, like at the same place we're calling log.bind, though I assume you were just putting it here as a way to test it